### PR TITLE
First

### PR DESCRIPTION
**Pull request by Lynn-Kish**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -7,29 +7,39 @@
...
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css">
+
 </head>
 <body>
+  <div id="time">
+    <h1>9:41</h1>
+</div>
+<i class="fas fa-network-wired common-icon"></i> 
+<i class="fas fa-wifi common-icon"></i>
+<i class="fas fa-battery-full common-icon"></i>
```
Comment (ciraganenicole) [inline index.html]: Remove these lines of code, they are not needed

File: index.html [hunk 2]
```diff
@@ -7,29 +7,39 @@
...
     <header>
         <h2>John Doe</h2>
 
         <nav>
           <ul class="menu">
-            <li><a href="#">Home</a></li>
-            <li><a href="#">About</a></li>
-            <li><a href="#">Contacts</a></li>
+            <li><a href="#"><i class="fas fa-bars"></i></a></li>
           </ul>
```
Comment (ciraganenicole) [inline index.html]: Replace the ul tag with div, this is not a list so no need to use ul, just use div and a tags

File: index.html [hunk 3]
```diff
@@ -7,29 +7,39 @@
...
+    <div class="container">
+      <svg>
+        <line x1="0" y1="5" x2="100" y2="5" />
+      </svg>
+    </div>
    
-        <footer class="footer section bd-container">
+        <section class="footer section bd-container">
           <div class="footer__container bd-grid">
               <div class="footer__content">
```
Comment (ciraganenicole) [inline index.html]: Change these classnames to suit the bloc purpose (remove footer)

**Review by ciraganenicole — CHANGES_REQUESTED** [review]
Hi @Lynn-Kish , 
Congratulations, you made a lot of progress but there are still some changes you have to make, and after you'll move to the next step

cap5

- Please respect margins and padding, especially in the first paragraph
- Align the skills items at the start, not in the center
- Increase the size of social's icons
- Change the color of the work button

**Comment by Lynn-Kish** [comment]
have made changes according to what u said ... hope u'll check on me as soon as possible

**Review by ciraganenicole — CHANGES_REQUESTED** [review]
Good work! But there are some changes you need to make:

- Titles, text, and buttons of the work section are aligned at the start, not the center
- Images are not visible, please, fix this

**Review by ciraganenicole — APPROVED** [review]
Hi @Lynn-Kish ,
Congratulations, you are good to go to the next step. You can merge this branch

**Comment by Lynn-Kish** [comment]
thx

**Pull request merged**